docs/rules/rules.tex

```diff
@@ -277,9 +277,8 @@ \section{Definitions}
   \end{record}
 
 \item[Chain] A chain is a sequence of blocks $b₁,…,b_n$ such that the
-  predecessor of $b₁$ is the genesis block, and for every $i$, the
-  predecessor of $b_{i+1}$ is $b_i$, and the predecessor of $b₁$ is
-  $b₀$.
+  predecessor of $b₁$ is the genesis block $b₀$, and for every $i$, the
+  predecessor of $b_{i+1}$ is $b_i$.
```

Comment:
Can you change « genesis block » into « genesis identifier »? (I'd normaly do it myself, but I'm on something else for the next few days). I'll merge then.

Reply:
Done in 654419b!